pkg/internal/checkup/trex/configfiles.go

```diff
@@ -170,3 +170,18 @@ func (c Config) GenerateExecutionScript() string {
 
 	return sb.String()
 }
+
+func GenerateSystemdUnitFile() string {
+	sb := strings.Builder{}
+
+	sb.WriteString("[Unit]\n")
+	sb.WriteString("Description=TRex Server 1\n")
```

Comment:
please change "TRex Server 1" to "TRex Server" - typo is in the original :)

Reply:
Done